src/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+  margin: 0;
+}
+
+.card-container {
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  font-family: Roboto, sans-serif;
+}
+
+.item-picture {
+  height: 134px;
+  width: 160px;
+  margin: 30px 19px;
+}
+
+.picture {
+  height: 100%;
+  width: 100%;
+}
+
+.item-title {
+  margin: 45px 14px 5px;
+  font-weight: 500;
+  line-height: 15px;
+  font-size: 12px;
+}
+
+.item-code {
+  margin-left: 14px;
+  font: 400 11px/13px Roboto, sans-serif;
```

Comment:
Don't repeat styles like font-family

Reply:
I changed this as well, I was trying to avoid using repeating but in font shorthand it must be there.